src/app/components/merge/MergeModal.js

```diff
@@ -124,7 +125,8 @@ const MergeModal = ({ contacts, contactID, userKeysList, ...rest }) => {
 MergeModal.propTypes = {
     contacts: PropTypes.arrayOf(PropTypes.array).isRequired,
     contactID: PropTypes.string,
-    userKeysList: PropTypes.array.isRequired
+    userKeysList: PropTypes.array.isRequired,
+    hasPaidMail: PropTypes.oneOfType([PropTypes.number, PropTypes.bool])
```

Comment:
It should be:
```suggestion
    hasPaidMail: PropTypes.bool
```
right?

Reply:
You're right!

Reply:
I took this from the `ContactRow` component, but after checking the `useUser` hook, I see that `hasPaidMail` is a number. I'll just transform it into a boolean through `!!hasPaidMail`